handlers/workflow_handler.go

```diff
@@ -144,20 +144,21 @@ func getWorkflows() gin.HandlerFunc {
 			prepareErrorResponse(err, c)
 			return
 		}
-		workflows, err := services.GetWorkflows(pageNumber, pageSize, sortByFields)
+		workflows, totalWorkflows, err := services.GetWorkflows(pageNumber, pageSize, sortByFields)
```

Comment:
Instead of totalWorkflows, can we rename it to totalWorkflowCount if it makes sense